readthedocs/search/tasks.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+from django.apps import apps
+from django_elasticsearch_dsl.registries import registry
+
+from readthedocs.worker import app
+
+log = logging.getLogger(__name__)
+
+
+def _get_index(indices, index_name):
+    for index in indices:
+        if str(index) == index_name:
+            return index
+
+
+def _get_document(model, document_class):
+    documents = registry.get_documents(models=[model])
+
+    for document in documents:
+        if str(document) == document_class:
+            return document
+
+
+@app.task(queue='web')
+def create_new_es_index_task(app_label, model_name, index_name, new_index_name):
+    model = apps.get_model(app_label, model_name)
+    indices = registry.get_indices(models=[model])
+    old_index = _get_index(indices=indices, index_name=index_name)
+    new_index = old_index.clone(name=new_index_name)
+    new_index.create()
+
+
+@app.task(queue='web')
+def switch_es_index_task(app_label, model_name, index_name, new_index_name):
```

Comment:
We don't put `task` at the end of our tasks.

Reply:
So what do we put to understand that its a task?

Reply:
We don't use any `suffix` in our code. We should be keeping the same coding standards here as elsewhere. If we want to change this, we need to change it everywhere, otherwise it's even more confusing having half of our tasks end in `task`.

Reply:
Yeah. Fixed it in latest commit